DuckDuckGo/Feedback/VPNMetadataCollector.swift

```diff
@@ -81,6 +81,7 @@ struct VPNMetadata: Encodable {
 
     func toBase64() -> String {
         let encoder = JSONEncoder()
+        encoder.outputFormatting = [.sortedKeys]
```

Comment:
This makes it so that the metadata in the feedback tasks is always consistently ordered.